docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_5.md

```diff
@@ -0,0 +1,52 @@
+# Audit of Sharing storage data access 
+
+Test verifies that any access to the data in the sharing object storages is logged.
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+
+**Preparations**:
+1. Login as the admin user from the prerequisites 
+2. Create storage `storage1` with ticked ***Enable sharing*** option
+3. In the `storage1`
+    - Add `file1` to the storage root
+    - Create `folder1` with 2 files (`inner_file1` and `inner_file2`)
+4. Click **Share** button
+5. Copy ***Share storage link*** to clipboard
+6. Logout
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open link stored at the step 5 of the Preparations. Login as the non-admin user from the prerequisites | |
+| 2 | Click **Upload** button | |
+| 3 | In pop-up window select a file (`file2`) and confirm uploading | |
+| 4 | Logout and _Check audit log_ (see below) | Record <br> `WRITE storage1/file2` <br> is shown in the top of logs records |
+| 5 | Repeat step 1 | |
+| 6 | Click **Download** button near the `file1` name | |
+| 7 | Logout and _Check audit log_ (see below) | Record <br> `READ storage1/file1` <br> is shown in the top of logs records |
+| 8 | Repeat step 1 | |
+| 9 |Click **Rename** button opposite `file1` | |
+| 10 | Specify a new value in the _Name_ field. Click **OK** button | |
+| 11 | Logout and _Check audit log_ (see below) | Records <br> `READ storage1/file1` <br> `WRITE storage1/file1_new` <br> `DELETE storage1/file1` <br> are shown in the top of logs records | 
+| 12 | Repeat step 1 | |
+| 13 | Click **Rename** button opposite `folder1` | |
+| 14 | Specify a new value in the _Name_ field. Click **OK** button | |
+| 15 | Logout and _Check audit log_ (see below) | For each files from `folder1` (*inner_file1* and *inner_file2*) records <br> `READ storage1/folder1/<file>` <br> `WRITE storage1/folder1_new/<file>` <br> `DELETE storage1/folder1/<file>` <br> are shown in the top of logs records | 
+| 16 | Repeat step 1 | |
+| 17 | Click **Remove** button near the `file2` name and confirm removing | |
+| 18 | Logout and _Check audit log_ (see below) | Record <br> `DELETE storage1/file2` <br> is shown in the top of logs records |
+| 19 | Repeat steps 1-2 | |
+| 20 | Click **Remove** button near the `folder1` name and confirm removing | |
+| 21 | Logout and _Check audit log_ (see below) | For each files from `folder1` (*inner_file1* and *inner_file2*) records <br> `DELETE storage1/folder1_new/<file>` <br> are shown in the top of logs records |
+
+
+\* _Check audit log_:
+1. Login as the admin user from the prerequisites | |
+2. Open the **Settings** page | |
+3. Open the **SYSTEM MANAGEMENT** page | |
+4. Select the **LOGS** tab | |
+5. Click ***Show advanced*** button | |
+6. Click **Type** field and select `audit` value from drop-down
```

Comment:
I suggest to add an action like "Clear the field **From**" - as by default logs are shown from the moment of the opening **Logs** form.